examples/tf/trpo_swimmer.py

```diff
@@ -22,7 +22,7 @@ def run_task(*_):
             baseline=baseline,
             max_path_length=500,
             discount=0.99,
-            step_size=0.01)
+            max_kl_step=0.01)
```

Comment:
Why is this change here?

Reply:
Because we renamed `step_size` to `max_kl_step` in the algorithm.

Reply:
Some parameters in `/examples` are outdated. Which one you think is unrelated change?

Reply:
it's true -- we changed this param to `max_kl_step` some time ago.